### ISSUE TITLE
optionExamples: Allow to pass JSON instead of a string

### ISSUE THREAD

**Issue by andy-hanson** [palantir/tslint#2467]
Current (taken from `arrowReturnShorthandRule.ts`):

```ts
optionExamples: [
    `[true]`,
    `[true, "${OPTION_MULTILINE}"]`,
],
```

Desired:

```ts
optionExamples: [
    true,
    [true, OPTION_MULTILINE],
],
```

We can just stringify the examples. This makes it easier to read (for the tslint contributor, in the documentation it will be the same), and the compiler will catch typos.

**Comment by andy-hanson** [comment]
This can be closed now.
